ui/README.md

```diff
@@ -72,3 +72,15 @@ From here you should be able to view the project at &lt;your-local-maas-ip>:8000
 When proposing changes to maas-ui, ensure you update CHANGELOG.md appropriately, describing your changes under either a "Fixed", "Added", "Removed", or "Changed" subheading under "Unreleased".
 
 See [keep a changelog](https://keepachangelog.com/en/1.0.0/) for details.
+
+# Release Process
+
+1. Run `yarn clean`.
+2. Run `yarn install`.
+3. Ensure tests pass with `yarn test`.
```

Comment:
Should steps 1-3 be part of `yarn build`? Or possibly a new `yarn release` command?